rules/int_crawler.go

```diff
@@ -133,7 +134,9 @@ func (ic *intCrawler) run() {
 			}
 		}
 		logger.Info("Crawler run complete")
-		for i := 0; i < ic.interval; i++ {
+		intervalSeconds := int(ic.interval.Generate().Seconds())
```

Comment:
If the jitter is only a fraction of a second, won't this result in either the base value or one less?

Reply:
That's right. I considered replacing this loop with a channel driven design instead which would allow instant reaction to a shutdown or nanosecond level granularity on the sleep time. I figured you all might object to the additional complexity 😅 so I opted for a small augmentation.

With this simpler design we still lock to the closest second, so a 10% jitter will only have an effect if the crawler interval is set to >10 via the option. I suspect most users will use a very large interval so they should be unaffected.